arrowhead/arrowhead.rb

```diff
@@ -15,20 +14,14 @@ class Arrowhead
     },
   }
 
-  # FIXME: I don't have time to deal with this.
-  def self.classify(region, shape)
-    if CLASSIFICATIONS.include? region
-      shapes = CLASSIFICATIONS[region]
-      if shapes.include? shape
-        arrowhead = shapes[shape]
-        "You have a(n) '#{arrowhead}' arrowhead. Probably priceless."
-      else
-        raise "Unknown shape value. Are you sure you know what you're talking about?"
-      end
-    else
-      raise "Unknown region, please provide a valid region."
+  def self.classify(region, test_shape)
+    possible_shapes = CLASSIFICATIONS[region]
+    raise "Unknown region, please provide a valid region." if possible_shapes == nil
+    if this_shape = possible_shapes.include?(test_shape)
+      "You have a(n) '#{this_shape}' arrowhead. Probably priceless."
```

Comment:
This also no longer raises the error "Unknown shape value. Are you sure you know what you're talking about?"